libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -1006,7 +1006,7 @@ void AP_TECS::update_pitch_throttle(int32_t hgt_dem_cm,
     _DT = (now - _update_pitch_throttle_last_usec) * 1.0e-6f;
     _update_pitch_throttle_last_usec = now;
 
-    _flags.is_gliding = _flags.gliding_requested || _flags.propulsion_failed || aparm.throttle_max==0;
+    _flags.is_gliding = _flags.gliding_requested || _flags.propulsion_failed || aparm.throttle_max==0 || _options & OPTION_GLIDER_ONLY;
```

Comment:
this is technically correct, although I'd muuuch rather see it with explicit ( ).

Reply:
Thanks, yes that is better!